### ISSUE TITLE
Doubt in Entropy Loss

### ISSUE THREAD

**Issue by sainivedh19pt** [jariasf/GMVAE#4]
Here the prob-cat/true_labels are obtained from [predicted logits](https://github.com/jariasf/GMVAE/blob/c39bdfac4163da551bd3bf5f37666a8a4aef2cd6/pytorch/networks/Layers.py#L68). How is this considered as true labels in [loss function](https://github.com/jariasf/GMVAE/blob/c39bdfac4163da551bd3bf5f37666a8a4aef2cd6/pytorch/losses/LossFunctions.py#L95)

Any help would be much appreciated. Thanks

**Comment by sainivedh19pt** [comment]
My bad, we are calculating entropy p*log(p). It make sense now.

**Comment by Sadegh-Saberian** [comment]
Hello,

thanks for the implementation. I still dont get the notation of target in the [entropy function](https://github.com/jariasf/GMVAE/blob/c39bdfac4163da551bd3bf5f37666a8a4aef2cd6/pytorch/losses/LossFunctions.py#:~:text=def-,entropy,-(self%2C%20logits%2C%20targets). Arent the logits and targets essentially the same thing for the loss calculation? One is the softmax of the other one. Some help on this is much appreciated. thanks!
